src/components/molecules/Wallet/Details.tsx

```diff
@@ -23,6 +44,14 @@ export default function Details(): ReactElement {
     setProviderInfo(providerInfo)
   }, [web3Provider])
 
+  useEffect(() => {
+    if (!networkId) return
+    console.log('networks', networksList, networkId)
```

Comment:
we can remove this `console.log`, or put it behind `Logger`

Reply:
busted